### DISCUSSION TITLE
目标消失，confidence还是挺大， 大于0.9

### DISCUSSION THREAD

**Discussion by StrugglingForBetter** [STVIR/pysot#457] [Q&A]
有时候，即使目标消失了，会框柱画面中的背景，返回的confidence还是蛮大的，甚至可以达到0.9的confidence

**Comment by StrugglingForBetter** [comment]
补充，还有时候, confidence比较小，但是却框柱了真实的目标，所以基于以上两种情况，如何确定tracking成功或失败？似乎有点难

**Comment by ZhiyuanChen** [comment]
confidence与是否跟住目标呈正相关，但没有绝对的连接

**Reply by StrugglingForBetter** [reply]
那请问如何表征追踪失败？可以详细说说吗？谢谢！

**Reply by ZhiyuanChen** [reply]
> 那请问如何表征追踪失败？可以详细说说吗？谢谢！

没有一个好的办法
